.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 23.2.6
-elixir 1.11.3
+elixir 1.18.2-otp-27
+erlang 27.0.1
```

Comment:
Is it necessary to update the Elixir and Erlang to upgrade the library? If yes, could you please update all other places using this old Elixir/Erlang version in the project?

Reply:
Good question. I'll make some tests to figure the minimal versions out and I'll update this.

Reply:
All right! I looked it up and turns out Elixir Make works for Elixir 1.9 and up. It wasn't working for me because older versions of OTP require some SSL-related shenanigans. I removed the changes to `.tool-versions`, since they are not needed for this change.

I would recommend completely removing this file and instead relying on `mix.exs` to enforce the Elixir and OTP versions instead, but this is out of the scope of this PR.

Reply:
Update: I also slightly change the `mix.exs` to stop this warning:
```

Run mix compile --warnings-as-errors
    warning: using single-quoted strings to represent charlists is deprecated.
    Use ~c"" if you indeed want a charlist or use "" instead.
    You may run "mix format --migrate" to change all single-quoted
    strings to use the ~c sigil and fix this warning.
    │
  4 │   if :erlang.system_info(:otp_release) < '21' do
    │                                          ~
    │
    └─ /home/runner/work/console/console/deps/gen_magic/mix.exs:4:42
    ```